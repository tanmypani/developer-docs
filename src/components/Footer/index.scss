@import '../../assets/styles/variables.scss';

footer {
    width: 100%;
    background: var(--body-background-color);

    .footerWrapper {
        text-align: center;
        color: #777e8b;
        font-size: 12px;
        font-weight: 300;
        margin-top: 50px;
    }
}
