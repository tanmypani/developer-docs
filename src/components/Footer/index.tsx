import React from 'react';
import t from '../../utils/lang-utils';
import './index.scss';

const Footer = () => (
    <footer>
        <section className="containerWrapper">
            <div className="footerWrapper">
                <span data-testid="footer-text">{t('COPYRIGHT_TEXT')}</span>
            </div>
        </section>
    </footer>
);

export default Footer;
