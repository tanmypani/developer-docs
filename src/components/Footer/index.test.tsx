import React from 'react';
import { render } from '@testing-library/react';

import Footer from './index';

jest.mock('../../utils/lang-utils', () => (key) => key);

describe('Footer', () => {
  it("should renders correctly", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot()
  })
})