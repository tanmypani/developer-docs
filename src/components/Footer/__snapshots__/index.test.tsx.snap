// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should renders correctly 1`] = `
<div>
  <footer>
    <section
      class="containerWrapper"
    >
      <div
        class="footerWrapper"
      >
        <span
          data-testid="footer-text"
        >
          COPYRIGHT_TEXT
        </span>
      </div>
    </section>
  </footer>
</div>
`;
